inc/oapv_config.h

```diff
@@ -32,7 +32,7 @@
 #ifndef __OAPV_CONFIG_H__9876543210123456789012345678901234567890123456789__
 #define __OAPV_CONFIG_H__9876543210123456789012345678901234567890123456789__
  
-// #define OAPV_LIBVERSION_HEADER // defined if the file oapv_libversion.h exists
-// #define OAPV_EXPORT_HEADER     // defined if the file oapv_exports.h exists
+#define OAPV_VERSION_HEADER // defined if the file oapv_libversion.h exists
+#define OAPV_EXPORT_HEADER  // defined if the file oapv_exports.h exists
```

Comment:
What is the point of these defines? Why would those headers not exist if they are unconditionally installed?

Without this you'd be able to remove the ifdeffery in src/oapv.c and allow oapv_version() to safely return a const char*. And the reason a const char* is better for it than an int is because ideally you'd fetch the project version using git describe --tags first, and if not possible (Like in a tarball snapshot), then version.txt would be parsed.